pkg/auth.go

```diff
@@ -236,39 +224,39 @@ func printQrCode(qrcode string) {
 
 // NewContractByType returns a ContractTemplate of a certain type, language and version.
 // If for the combination of type, version and language no contract can be found, the error is of type ErrContractNotFound
-func (auth *Auth) ContractByType(contractType ContractType, language Language, version Version) (*ContractTemplate, error) {
-	return NewContractByType(contractType, language, version, auth.ValidContracts)
+func (auth *Auth) ContractByType(contractType contract.ContractType, language contract.Language, version contract.Version) (*contract.ContractTemplate, error) {
```

Comment:
why do we have Language and Version in `contract` package, but `Type` if `ContractType`? (prefixed with `Contract`)

Reply:
The pkt.Type is very ambigious (contrary to Language), contract.Type is better. We now have the ability to change that.